src/server/models/list.ts

```diff
@@ -44,25 +44,21 @@ export async function findListByCountryAndType(country: CountryName, type: Servi
   }
 }
 
-export async function findListByAnnualReviewDate(annualReviewStartDate: Date, listItemStatuses: Status[] = [], isAnnualReview?: boolean): Promise<Result<List[]>> {
+export async function findListByAnnualReviewDate(annualReviewStartDate: Date, today: Date, listItemStatuses: Status[] = [], isAnnualReview?: boolean): Promise<Result<List[]>> {
   try {
     logger.debug(`searching for lists matching date [${annualReviewStartDate}]`);
 
     const result = (await prisma.list.findMany({
       where: {
-        nextAnnualReviewStartDate: annualReviewStartDate,
-        isAnnualReview: false,
```

Comment:
is there a reason why you pass in `isAnnualReview` rather than just querying by isAnnualReview? do you use it anywhere?

Reply:
nope, not since refactoring.  These have now been removed.